test/controllers/genres_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
You should delete this file, since you already have `spec/controllers/api/genres_controller_spec.rb`